src/main/java/de/codingair/tradesystem/trade/Trade.java

```diff
@@ -232,6 +232,14 @@ private void finish() {
         Runnable runnable = new Runnable() {
             @Override
             public void run() {
+                // code to avoid some weird money dupe
+                Profile p0 = TradeSystem.getProfile(players[0]);
+                Profile p1 = TradeSystem.getProfile(players[1]);
+                if (p0.getMoney() > money[0] || p1.getMoney() > money[1]) {
+                    cancel(Lang.getPrefix() + Lang.get("Economy_Error"));
+                    return;
```

Comment:
Leaves the GUI open.

Reply:
The GUI is closed by the cancel method (line 239)
![image](https://user-images.githubusercontent.com/52083283/91763672-c87ef880-eb8a-11ea-8998-239e0bfb5e6a.png)

Reply:
Haha, I'm blind :D Sorry. 
BUT one thing left. `p0.getMoney() > money[0]` is wrong. This triggers an Economy_Error if the player has more money  than his offer.

Reply:
Fixed